mobify/platform-scaffold
# AMP

The big `amp` -> `develop` merge

## Changes
- Adding `amp/`
- Updating the web project (mostly connector code) to be AMP-compatible

## Todos
~~- [ ] Add a high-level description of your changes to the CHANGELOG.md, including a link to the PR with your changes~~

- [x] Get #867 merged
- [x] Get #1008 merged

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Make sure everything works as before
- `cd ../amp` run `npm run dev`
- Check out http://localhost:3000/potions.html
